projectile.el

```diff
@@ -1116,6 +1117,12 @@ https://github.com/emacs-helm/helm")))
           (grizzl-completing-read prompt (grizzl-make-index choices))
         (user-error "Please install grizzl from \
 https://github.com/d11wtq/grizzl")))
+     ((eq projectile-completion-system 'ivy)
+      (if (fboundp 'ivy-read)
+          (ivy-read prompt (cl-delete-duplicates choices
+                                                 :test #'equal) initial-input)
+        (user-error "Please install ivy from \
+https://github.com/abo-abo/swiper")))
```

Comment:
> Why don't you spin ivy as a separate project?

It's separate in MELPA. But since I'm still figuring out the responsibilities for each package, they
are in the same repository (easier to refactor that way).